Jenkinsfile

```diff
@@ -49,9 +49,12 @@ pipeline {
     stage('Check indentation') {
       steps {
         sh './doc/indent'
-        sh 'git diff | tee changes-astyle.diff'
+        sh 'git diff > changes-astyle.diff'
         archiveArtifacts artifacts: 'changes-astyle.diff', fingerprint: true
-        sh 'git diff --exit-code --name-only'
+        sh '''
+	  git diff --exit-code || \
+	  { echo "Please check indentation, see artifacts in the top right corner!"; exit 1; }
```

Comment:
Yes, this now looks good in the jenkins test: https://jenkins.tjhei.info/blue/organizations/jenkins/aspect/detail/PR-2326/4/pipeline

Please change the "see artifacts..." text and remove the test error in the .cc file again.

Reply:
Do you want this text removed or do you want me to rephrase it?

Reply:
Your choice, I just don't want it to be factually wrong :-)

Reply:
But the text is correct, we archive the artifacts (the changes diff), and they are available in the top right corner under `Artifacts` for download.

Reply:
Ah, I see. Thanks.